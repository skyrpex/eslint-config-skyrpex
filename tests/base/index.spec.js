const { createEngine, cleanReport } = require("../utils");

const engine = createEngine({
    configFile: require.resolve("./eslint.config.js"),
});

test("should report missing trailing slash", () => {
    const report = engine.executeOnFiles(["missing-trailing-slash.fixture.js"]);
    expect(cleanReport(report)).toMatchSnapshot();
});
