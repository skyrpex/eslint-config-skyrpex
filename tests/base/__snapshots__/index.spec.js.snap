// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should report missing trailing slash 1`] = `
Object {
  "errorCount": 1,
  "fixableErrorCount": 1,
  "fixableWarningCount": 0,
  "results": Array [
    Object {
      "errorCount": 1,
      "filePath": "tests/fixtures/missing-trailing-slash.fixture.js",
      "fixableErrorCount": 1,
      "fixableWarningCount": 0,
      "messages": Array [
        Object {
          "column": 27,
          "endColumn": 27,
          "endLine": 6,
          "fix": Object {
            "range": Array [
              148,
              148,
            ],
            "text": ",",
          },
          "line": 6,
          "message": "Insert \`,\`",
          "nodeType": null,
          "ruleId": "prettier/prettier",
          "severity": 2,
          "source": "    testVeryLongMethodName",
        },
      ],
      "source": "const testVeryLongMethodName = () => {};

testVeryLongMethodName(
    testVeryLongMethodName,
    testVeryLongMethodName,
    testVeryLongMethodName
);
",
      "warningCount": 0,
    },
  ],
  "warningCount": 0,
}
`;
